Sources/Commandant/Option.swift

```diff
@@ -193,6 +193,47 @@ public func <| <T: ArgumentProtocol, ClientError>(mode: CommandMode, option: Opt
 	}
 }
 
+/// Evaluates the given option in the given mode.
+///
+/// If parsing command line arguments, and no value was specified on the command
+/// line, `nil` is used.
+public func <| <T: ArgumentProtocol, ClientError>(mode: CommandMode, option: Option<[T]?>) -> Result<[T]?, CommandantError<ClientError>> {
```

Comment:
Yep, makes sense. I've created one in 449fcc1. I was following what was existed for non-array version. Seems reasonable to me.